bpfd/src/oci_utils/image_manager.rs

```diff
@@ -493,7 +494,9 @@ fn load_image_meta(image_content_path: PathBuf) -> Result<ContainerImageMetadata
     let file_content =
         read_to_string(image_config_path).context("failed to read image config file")?;
 
-    let image_config: Value = serde_json::from_str(&file_content).unwrap();
+    // This should panic since it means that the on disk memory format has changed during runtime.
```

Comment:
on disk memory? you mean on disk storage format?

Reply:
Lol yerp thanks!